lib/cache_manager.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:io';
+
+import 'package:flutter_cache_manager/flutter_cache_manager.dart';
+import 'package:path_provider/path_provider.dart';
+
+class CustomCacheManager {
+  static const key = 'libCachedImageData';
+  static final _repo = CacheObjectProvider(databaseName: key);
+  static final instance = CacheManager(
+    Config(
+      key,
+      stalePeriod: const Duration(days: 30),
+      maxNrOfCacheObjects: 500,
```

Comment:
500 for `maxNrOfCacheObjects` seems too small, considering there can be hundreds of emotes and/or badges for each channel.

What do you think about 10,000? Seems like a good number to start with based on my rough estimate here: considering a 1GB image cache limit, if we assume an emote size of 100kb, 1,000,000 KB / 100 KB = 10,000 emotes. 7TV 4x emotes seem to be around 10-30 kb while GIFs are around 100-300 kb.

```suggestion
      maxNrOfCacheObjects: 10000,
```

Reply:
I think you're right, 500 is too small. I mean the most important part is clearing images which wasn't used for more than 30 days.